server/cyf-data.sql

```diff
@@ -0,0 +1,74 @@
+drop table if exists students cascade;
+drop table if exists mentors cascade;
+drop table if exists region;
+drop table if exists learningobjectives cascade;
+drop table if exists mappingskills cascade;
+drop table if exists techskills cascade;
+drop table if exists competencylevels cascade;
+​
+CREATE TABLE users (
```

Comment:
Are we replacing this with students, I mean students changes to users, if yes, then we need to insert into users below